demisto_sdk/commands/content_graph/parsers/content_item.py

```diff
@@ -79,15 +83,24 @@ def __init__(
         self,
         path: Path,
         pack_marketplaces: List[MarketplaceVersions] = list(MarketplaceVersions),
+        marketplace: Optional[MarketplaceVersions] = None,
     ) -> None:
         self.pack_marketplaces: List[MarketplaceVersions] = pack_marketplaces
+        self.marketplace = marketplace
         super().__init__(path)
         self.relationships: Relationships = Relationships()
 
+    def validate_marketplace(self):
+        if self.marketplace and self.marketplace not in self.marketplaces:
+            raise NotInMarketplaceException(
+                f"{self.node_id} is not in {self.marketplace}, only in {self.marketplaces}"
```

Comment:
my suggestion following our discussion:
```suggestion
    def validate_marketplace(self, current_marketplace: Optional[MarketplaceVersions]):
        if current_marketplace and current_marketplace not in self.marketplaces:
            raise NotInMarketplaceException(
                f"{self.node_id} is not in {current_marketplace}, only in {self.marketplaces}"
```
then you won't need the new marketplace attribute